### PR TITLE
CON-847: Publish Dokka to artifactory

### PR DESCRIPTION
**Pull request by anien**
Added notes about the changes made to Dokka, update README.md
Updated the Maven group to com.r3.conclave.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Project Settings
-dokka_version_base=1.5.31
-dokka_publication_channels=maven-central-snapshot&space-dokka-dev
+#dokka_version_base=1.5.31
```
Comment (filipesoliveira) [inline gradle.properties]: The commented out code can be removed.

**Comment by filipesoliveira** [comment]
LGTM. I made a suggestion but I am happy to approve it without it.

File: gradle.properties (hunk 1 above)
Comment (anien) [reply gradle.properties]: Fixed!

**Review by filipesoliveira — APPROVED** [review]

**Comment by filipesoliveira** [comment]
It might be a good idea to "squash and merge" this PR.

**Pull request merged**